powerline/segments/i3wm.py

```diff
@@ -16,12 +16,12 @@ def calcgrp(w):
 	return group
 
 
-def workspaces(pl):
+def workspaces(pl, strip=0):
 	'''Return workspace list
 
 	Highlight groups used: ``workspace``, ``w_visible``, ``w_focused``, ``w_urgent``
 	'''
 	return [{
-		'contents': w['name'],
+		'contents': w['name'][min(len(w['name']),strip):],
```

Comment:
It is rather strange to see stripping like `my workspace` → `y workspace` → ` workspace` → …: usually end of the name is stripped.

Also note that in place of stripping format argument is usually provided: e.g. `fmt.format(**w)` with default `'{name}'`.